core/src/test/scala/org/apache/spark/storage/BlockManagerSuite.scala

```diff
@@ -1932,22 +1932,29 @@ class BlockManagerSuite extends SparkFunSuite with Matchers with BeforeAndAfterE
     assert(master.getLocations(blockIdLarge) === Seq(store1.blockManagerId))
   }
 
-  test("test migration of shuffle blocks during decommissioning") {
-    val shuffleManager1 = makeSortShuffleManager()
+  private def testShuffleBlockDecommissioning(maxShuffle: Option[Int], migrate: Boolean) = {
```

Comment:
Maybe rename "migrate" to "willReject"?

All cases actually will try to migrate the blocks first. The difference is whether it'll be rejected or not.

Reply:
Sure